shared/database/phd.go

```diff
@@ -0,0 +1,87 @@
+package database
+
+import (
+	"context"
+
+	"github.com/Pocket/global-services/shared/utils"
+	dbclient "github.com/pokt-foundation/db-client/client"
+	"github.com/pokt-foundation/pocket-go/provider"
+	"github.com/pokt-foundation/portal-db/types"
+	"golang.org/x/exp/slices"
+)
+
+// PostgresDBClient holds the phd client lib and additional methods for filtering applications
+type PostgresDBClient struct {
+	*dbclient.DBClient
+}
+
+// NewPHDClient returns an augmented HTTP client for the pocket http db which adheres which adds app filters
+func NewPHDClient(config dbclient.Config) (*PostgresDBClient, error) {
+	client, err := dbclient.NewDBClient(config)
```

Comment:
See my comment above about calling the `NewReadOnlyDBClient` instead of `NewDBClient` since this is where the func is actually called.